code/modules/surgery/organs/robolimbs.dm

```diff
@@ -3,142 +3,219 @@ GLOBAL_LIST_EMPTY(chargen_robolimbs)
+	// If 0, object is a model. If 1, object is a brand (that serves as the default model) with child models. If 2, object is a brand that has no child models and thus also serves as the model..
+	var/has_subtypes = childless
 	var/parts = list("chest", "groin", "head", "r_arm", "r_hand", "r_leg", "r_foot", "l_leg", "l_foot", "l_arm", "l_hand")	// Defines what parts said brand can replace on a body.
 
+/* Arksoft's Tableau Entertainment */
+/datum/robolimb/tableau
+	company = "Tableau Entertainment Powered Mechanics"
+	desc = "This limb is made from a cheap polymer."
+	icon = 'icons/mob/human_races/cyberlimbs/nanotrasen/nanotrasen_main.dmi'
+	has_subtypes = childless
+	is_monitor = TRUE
+
+/* Bishop */
+//Main
 /datum/robolimb/bishop
 	company = "Bishop Cybernetics"
 	desc = "This limb has a white polymer casing with blue holo-displays."
 	icon = 'icons/mob/human_races/cyberlimbs/bishop/bishop_main.dmi'
-	has_subtypes = 1
+	has_subtypes = brand
 
-/datum/robolimb/bishop/alt1
-	company = "Bishop Cybernetics alt."
+/datum/robolimb/bishop/monitor
+	company = "Bishop Knight"
 	icon = 'icons/mob/human_races/cyberlimbs/bishop/bishop_alt1.dmi'
 	parts = list("head")
-	selectable = 0
-	has_subtypes = null
+	has_subtypes = model
+	is_monitor = TRUE
 
-/datum/robolimb/bishop/monitor
-	company = "Bishop Cybernetics mtr."
+
+//Rook
+/datum/robolimb/rook
+	company = "Bishop Rook"
+	desc = "This limb has a polished metallic casing and a holographic face emitter."
+	icon = 'icons/mob/human_races/cyberlimbs/bishop/bishop_rook.dmi'
+	has_subtypes = brand
+
+/datum/robolimb/rook/monitor
+	company = "Bishop Castle"
 	icon = 'icons/mob/human_races/cyberlimbs/bishop/bishop_monitor.dmi'
 	parts = list("head")
-	is_monitor = 1
-	selectable = 0
-	has_subtypes = null
+	has_subtypes = model
+	is_monitor = TRUE
 
+/* Hesphiastos */
+//Main
 /datum/robolimb/hesphiastos
 	company = "Hesphiastos Industries"
 	desc = "This limb has a militaristic black and green casing with gold stripes."
 	icon = 'icons/mob/human_races/cyberlimbs/hesphiastos/hesphiastos_main.dmi'
-	has_subtypes = 1
-
-/datum/robolimb/hesphiastos/alt1
-	company = "Hesphiastos Industries alt."
-	icon = 'icons/mob/human_races/cyberlimbs/hesphiastos/hesphiastos_alt1.dmi'
-	parts = list("head")
-	is_monitor = 1
-	selectable = 0
-	has_subtypes = null
+	has_subtypes = brand
 
 /datum/robolimb/hesphiastos/monitor
-	company = "Hesphiastos Industries mtr."
+	company ="Industrial Revolution"
 	icon = 'icons/mob/human_races/cyberlimbs/hesphiastos/hesphiastos_monitor.dmi'
 	parts = list("head")
-	is_monitor = 1
-	selectable = 0
-	has_subtypes = null
+	has_subtypes = model
+	is_monitor = TRUE
+
+//Titan
+/datum/robolimb/titan
+	company = "Hesphiastos Titan"
+	desc = "This limb has a casing of an olive drab finish, providing a reinforced housing look."
+	icon = 'icons/mob/human_races/cyberlimbs/hesphiastos/hesphiastos_titan.dmi'
+	has_subtypes = brand
+
+/datum/robolimb/titan/monitor
+	company = "Titan Enforcer"
```

Comment:
Why don't the /monitors inherit "company" from their parent datums?

Because the parent datums have their own monitors in their dmi files.